intro/Readme.md

```diff
@@ -1,29 +1,86 @@
-1) What are the three most essential skills that a beginner programmer needs to know?
```

Comment:
This file should not be edited as part of this PR.  You should be creating/editing chapter_3/Readme.md.